model_prices_and_context_window.json

```diff
@@ -112,5 +112,25 @@
     "together-ai-40.1b-70b": {
         "input_cost_per_token": 0.000003,
         "output_cost_per_token": 0.000003
-    }
+    },
```

Comment:
we already have claude see line 42

Reply:
we're missing some vertex llms tho 

I'd recommend seeing llms in __init__.py to see what's missing